packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -175,7 +184,7 @@ export const FormFeedback = (props: FormFeedbackProps) => {
           ref={calculateHeight}
           sx={getFormTextAnimation(!!props.informationText && !props?.successfulText && !props.warningText)}
         >
-          {informationMessage}
+          {addFullStop(informationMessage)}
```

Comment:
❓ Why don't we update the localized messages ?

Reply:
@panteliselef we can't do this, or at least it would be very error-prone if we do. 
For example, the warning/error will be a combination of these keys which. happens dynamically.
```   passwordComplexity: {
      sentencePrefix: 'Your password must contain',
      minimumLength: '{{length}} or more characters',
      maximumLength: 'less than {{length}} characters',
      requireNumbers: 'a number',
      requireLowercase: 'a lowercase letter',
      requireUppercase: 'an uppercase letter',
      requireSpecialCharacter: 'a special character',
    }, 

I think it is better to do this in the end. Here I wrapped all messages with the helper purely for convention and possible future refactors.

Reply:
You are correct, I missed this one.
In that case, because we always want to append the full-stop in the end, consider updating the `generateErrorTextUtil` function that generates this string.
This will enable you to remove the custom function for adding a full-stop.

Reply:
@panteliselef If we move this to `generateErrorTextUtil` we will lose the ability to apply this throughout the app, something I think we want in general. WDYT?

Reply:
I don't think it's safe to assume that we can add full stops like this. This should be something included in a localized value. 
For the case of Password Complexity errors we can add a full-stop "by hand" the moment that we construct the string. Same for password strength errors. These two are the only strings generated by concatinating other localized values.